services/meta/client.go

```diff
@@ -305,7 +305,7 @@ func (c *Client) Database(name string) (*DatabaseInfo, error) {
 		}
 	}
 
-	return nil, nil
+	return nil, fmt.Errorf("Database '%s' does not exist", name)
```

Comment:
nit: all errors should be lower case so if they are wrapped in another error they read ok.

```
return nil, fmt.Errorf("database '%s' does not exist", name)
```